extensions/blocks/contact-info/address/edit.native.js

```diff
@@ -35,83 +42,86 @@ class AddressEdit extends Component {
 			},
 			setAttributes,
 			onFocus,
+			getStylesFromColorScheme,
 		} = this.props;
 
 		const externalLink = (
-			<ToggleControl
-				label={ __( 'Link address to Google Maps', 'jetpack' ) }
-				checked={ linkToGoogleMaps }
-				onChange={ newlinkToGoogleMaps => {
-					setAttributes( { linkToGoogleMaps: newlinkToGoogleMaps } );
-					onFocus();
-				} }
-			/>
+			<InspectorControls>
+				<PanelBody title={ __( 'Contact Info Settings', 'jetpack' ) }>
+					<ToggleControl
+						label={ __( 'Link address to Google Maps', 'jetpack' ) }
+						checked={ linkToGoogleMaps }
+						onChange={ newlinkToGoogleMaps => {
+							setAttributes( { linkToGoogleMaps: newlinkToGoogleMaps } );
+							onFocus();
+						} }
+					/>
+				</PanelBody>
+			</InspectorControls>
 		);
 
+		const textData = [
+			{
+				value: address,
+				placeholder: __( 'Street Address', 'jetpack' ),
+				onChange: newAddress => setAttributes( { address: newAddress } ),
+			},
```

Comment:
Nice improvement! 👍

Reply:
Thanks! I further refactored by generalizing the onChange function and just specifying the key of the attribute to set, but I can be convinced to keep it as is as well. I can see both ways.